tools/subgraph-example/hardhat.config.ts

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable prettier/prettier */
+import * as dotenv from "dotenv";
+import * as fs from 'fs';
+import * as path from 'path';
+import "@typechain/hardhat";
+import "@nomiclabs/hardhat-waffle";
+import "@graphprotocol/hardhat-graph";
+import { task } from "hardhat/config";
+import { mintNFT, transferERC20, createGravatar } from "./scripts";
+
+dotenv.config();
+
+task("deployERC20", "Deploys ERC20 contract", async (taskArgs, hre) => {
+  const contractName = "ExampleERC20";
+  await hre.run("compile");
+
+  const wallet = createWallet(hre);
+  const ERC20 = await hre.ethers.getContractFactory(
+    contractName,
+    wallet
+  );
+  const contract = await ERC20.deploy(20);
+
+  await contract.deployed();
+  const deployTx = await contract.deployTransaction.wait();
+  const address = deployTx.contractAddress;
+
+  await hre.run("graph", { contractName, address });
+  updateStartBlock(contractName, deployTx.blockNumber, hre);
+
+  console.log("CONTRACT DEPLOYED AT:")
+  console.log(address);
+  console.log(contract.address);
```

Comment:
nit: Those are the same in my experience. Same with the ones in ERC721 and Gravatar.

Reply:
Not on the mirror node version used in the custom config, because the latest mirror node image does not include an important fix, so I can't currently use it.

Reply:
Yep, I figured that the reason was mirror-node version. We can revisit this, when the fix is released.

Reply:
@Nana-EC https://github.com/hashgraph/hedera-mirror-node/issues/5005 and the PR https://github.com/hashgraph/hedera-mirror-node/pull/5006, but the fix did not get into 0.70.1